product_docs/docs/biganimal/release/using_cluster/tagging/index.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: Tagging BigAnimal resources
+---
+
+BigAnimal provides a shared tags system that allows you to assign and manage tags for resources across different resource types.
+
+The tags are assigned to the following resource types:
+
+- Project 
+- Cluster
+
+Key features of shared tags system are:
+
+- **Shared tags** &mdash; You can [create shared tags](./create_and_manage_tags.mdx) that are accessible and applicable to all the resource types. Shared tags provides a standard way to categorize and organize resources. Tags are scoped to the organizations and shared among all users in the organization. The access to tags is controlled by existing permission system.
+
+- **Tag assignment** &mdash; You can assign one or more shared tags to individual resource items. This capability enables you to categorize resources effectively and according to the needs.
+
+- **Resource filtering** &mdash; You can filter resources based on shared tags. This significantly improves the efficiency of resources retrieval, allowing to search related resources regardless of the resource type.
```

Comment:
for resource filtering and tag search: do we need docs on how to filter by tags or search by tags?

Reply:
@naveedkarajgi @hfallife, I need inputs on this to add it to the docs

Reply:
removed this for now